buildconfig/download_win_prebuilt.py

```
@@ -102,12 +102,12 @@ def get_urls(x86=True, x64=True):
         '3786bc016d89ca4cf9739e4d342ca29e3e29c137'
         ],
         [
-        'https://github.com/libsdl-org/SDL_mixer/releases/download/release-2.8.1/SDL2_mixer-devel-2.8.1-VC.zip',
-        'ea302368159ca64056b2519b3a624525f625220e',
+        'https://github.com/libsdl-org/SDL_mixer/releases/download/release-2.8.2/SDL2_mixer-devel-2.8.2-VC.zip',
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)